plugins/fclrave/fclspace.cpp

```diff
@@ -101,12 +97,13 @@ void FCLSpace::ReloadKinBodyLinks(KinBodyConstPtr pbody, FCLKinBodyInfoPtr pinfo
                 if( !pfclgeom ) {
                     continue;
                 }
-                // because user data is null pointer here, in CheckNarrowPhaseGeomCollision report.pgeom ends up being null pointer and we lose the geometry information.
-                // maybe pass plink->GetGeometries()[index] where index=distance(vgeometryinfos.begin(), itgeominfo)?
-                // or, should the collision report store names of body, link, and geom?
-                // also, currently there is no information about which geometry group was used for collision checking.
+                // currently there is no information about which geometry group was used for collision checking.
```

Comment:
Is the geometry group the `pinfo->_geometrygroup` above? If so, _should_ we save it?

Reply:
thanks for your comment!

Not sure right now, but we might want in future.

This PR fixes the geometry name since it's already introduced in `CollisionInfoPair` and there is an user of it. In that sense, we should show the geometry name unconditionally.

For `geometry group`, there is no user of it to read it via `CollisionInfoPair` and we can investigate whether it needs to show in the `CollisionInfoPair` in future.
At least, I have no strong motivation to remove this comment at this moment.